build/config/build.yaml

```diff
@@ -82,4 +87,61 @@ steps:
   dir: 'tests/'
   waitFor: ['DOCKER_STAGE', 'DOCKER_BUILD_MVN_GCLOUD']
 
+#
+# End of App Engine Integration tests
+
+#
+# GKE Integration tests
+#
+
+# Create a Docker image with the test application
+- name: 'gcr.io/cloud-builders/docker'
+  id: 'DOCKER_BUILD_TEST_APPLICATION'
+  args: ['build', '--tag=${_STAGING_IMAGE}-test-app', '--no-cache', 'tests/runtimes-common-testing/target/deploy']
```

Comment:
I don't think so, actually. More work would be required to make this possible - we'd need to set up some new maven profiles to only execute the tests without needing to re-run the maven build.

In the short term, I think if you just modified the JETTY_SMOKE_TEST step to remove the `clean` phase, it should work fine, although it's not ideal. 

@meltsufin WDYT?